frontend/web/components/modals/CreateFlag.js

```diff
@@ -1335,11 +1341,24 @@ const CreateFlag = class extends Component {
                                                       invalid ||
                                                       !savePermission
                                                     }
+                                                    dropdownItems={[
+                                                      {
+                                                        label: releasePipelineId
+                                                          ? 'Remove from Release Pipeline'
+                                                          : 'Add to Release Pipeline',
+                                                        onClick:
+                                                          releasePipelineId
+                                                            ? this
+                                                                .removeFromReleasePipeline
+                                                            : this
+                                                                .openReleasePipelineModal,
+                                                      },
+                                                    ]}
                                                   >
                                                     {isSaving
                                                       ? 'Updating'
                                                       : 'Update Feature Value'}
-                                                  </Button>
+                                                  </ButtonDropdown>
```

Comment:
I would put a "+" icon personally. So for us the dropdown is way better I'd say because we know the flow but as for a new feature I would sacrifice the esthetic to have it really obvious.

Or a + to trigger a bit of curiosity of what can I add there? I'm not opiniated at all, just trying what would be the most obvious for a user